app/assets/stylesheets/components/page_specific/_retirements_view_all.scss

```diff
@@ -0,0 +1,11 @@
+.view-all__item--hidden {
```

Comment:
To follow BEM strictly, we normally only put classes in the file that are named the same, so instead of .view-all__item you would have .retirements_view_all__item...

Or change the file name to _view-all.scss.